### PR TITLE
Add Batch custom image build instructions

### PR DESCRIPTION
**Pull request by wenyhu-google**

### CODE REVIEW

File: build-custom-image/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Overview
+[Batch](https://cloud.google.com/batch/docs/get-started) has provided Batch images under project `batch-custom-image` for different OS types.
+The Batch images pre-installs the required packages for running Batch jobs, which can help you save Batch job running time, as well as overcoming network limitations.
+
+This folder provides strategy on how to build a Batch required (preferred) image with different OS types. Due to the latency on GitHub update, if you meet with any issues when running the scripts in the folder, feel free to get help from Batch team.
```
Comment (bolianyin) [inline build-custom-image/README.md:5]: "This folder contains instructions on how to create a Batch compatible image with various operating system types."

Remove "Due to ... get help from Batch team". We don't have an official channel for that.

File: build-custom-image/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Prerequisites
+
+1.  Make sure the Compute Engine API with the default Compute Engine service
```
Comment (bolianyin) [inline build-custom-image/README.md:9]: 1,2,3. All use "Make sure" or "Ensure" to be consistent.

File: build-custom-image/build-centos-image/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+## Overview
+Batch required CentOS Images can be built using Google supported [Packer](https://github.com/GoogleCloudPlatform/cloud-builders-community/tree/master/packer).
```
Comment (bolianyin) [inline build-custom-image/build-centos-image/README.md:2]: Batch *compatible* CentOS ...

File: build-custom-image/build-cos-image/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+## Overview
+Batch required Container-Optimized (COS) Images can be built with [cos-customizer](https://cos.googlesource.com/cos/tools/+/refs/heads/master/src/cmd/cos_customizer).
```
Comment (bolianyin) [inline build-custom-image/build-cos-image/README.md:2]: *compatible*

File: build-custom-image/build-debian-image/README.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
+## Overview
+Batch required Debian Images can be built using Google supported [Packer](https://github.com/GoogleCloudPlatform/cloud-builders-community/tree/master/packer).
```
Comment (bolianyin) [inline build-custom-image/build-debian-image/README.md:2]: ditto.

File: build-custom-image/README.md (hunk 1 above)
Comment (wenyhu-google) [reply build-custom-image/README.md:5]: Modified, thanks!

File: build-custom-image/README.md (hunk 2 above)
Comment (wenyhu-google) [reply build-custom-image/README.md:9]: All uses "Make sure" now.

File: build-custom-image/build-centos-image/README.md (hunk 3 above)
Comment (wenyhu-google) [reply build-custom-image/build-centos-image/README.md:2]: Done.

File: build-custom-image/build-debian-image/README.md (hunk 5 above)
Comment (wenyhu-google) [reply build-custom-image/build-debian-image/README.md:2]: Done.

File: build-custom-image/build-cos-image/README.md (hunk 4 above)
Comment (wenyhu-google) [reply build-custom-image/build-cos-image/README.md:2]: Done.

**Review by bolianyin — APPROVED** [review]

**Pull request merged**